pkg/apiserver/apiserver.go

```diff
@@ -116,14 +124,26 @@ const (
 	MaxTimeoutSecs = 600
 )
 
+type staticLister struct {
```

Comment:
At other places we add `var _ APIResourceLister := &staticLister{}` to ensure and document that staticLister implements the APIResourceLister interface.